README.md

```diff
@@ -1,2 +1,74 @@
 # Awesome-Books
-This an app created using javascript
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+> This project is a book collection and shuffling application.
+
+## Description
+
+## Hosting
+
+This Portfolio is hosted on GitHub Page. [Click here](https://summyalena.github.io/Awesome-Books/)
+
```

Comment:
- [ ] I noticed that this link is broken, so kindly fix it or remove it for now 👍

Reply:
## Resolved 👍 

- removed the broken link
- no longer hosting this project
- thanks for noticing